modular_skyrat/modules/gunsgalore/code/ammo/ammo.dm

```diff
@@ -63,3 +71,175 @@
+	damage = 30
+	wound_bonus = 25
+	armour_penetration = 30
+	wound_falloff_tile = -5
+	fire_stacks = 2
+	leaves_fire_trail = FALSE
+
+/obj/item/ammo_casing/realistic/a762x39/xeno
+	name = "5.6x40mm anti-acid bullet casing"
+	desc = "A 5.6x40mm anti-acid bullet casing. Special chemical treatment and an additional layer of water-absorbent materials dissipates and absorbs water from the target's body, making any acid-blooded target melt from the inside."
+	extended_desc = "ANTI-ACID: Deals additional damage against advanced sentient Xenomorphs."
+	projectile_type = /obj/projectile/bullet/a762x39/hunting/xeno
+
+/obj/projectile/bullet/a762x39/xeno
+	name = "5.6mm anti-acid bullet"
+	damage = 25
+	wound_bonus = 15
+	armour_penetration = 0
+		/// Bonus force dealt against certain mobs
+	var/nemesis_bonus_force = 25
+		/// List of mobs we deal bonus damage to
+	var/list/nemesis_path = list(/mob/living/carbon/alien)
+
+/obj/projectile/bullet/a762x39/xeno/prehit_pierce(mob/living/target, mob/living/carbon/human/user)
+	if(istype(target, nemesis_path))
+		damage += nemesis_bonus_force
+	.=..()
+
+/obj/item/ammo_casing/realistic/a762x39/ap
+	name = "5.6x40mm armor-piercing bullet casing"
+	desc = "A 5.6x40mm armor-piercing bullet casing."
+	extended_desc = "ARMOR PIERCING: Increased armor piercing capabilities. What did you expect?"
+	projectile_type = /obj/projectile/bullet/a762x39
+
+/obj/projectile/bullet/a762x39/ap
+	name = "5.6mm armor-piercing bullet"
+	damage = 30
+	wound_bonus = 15
+	armour_penetration = 60
+	wound_falloff_tile = 0
+
+/obj/item/ammo_casing/realistic/a762x39/emp
+	name = "5.6x40mm ion bullet casing"
+	desc = "A 5.6x40mm ion bullet casing."
+	extended_desc = "EMP: Produces an Electro-Magnetic Pulse on impact, damaging electronics severely."
+	projectile_type = /obj/projectile/bullet/a762x39/emp
+
+/obj/projectile/bullet/a762x39/emp
+	name = "5.6mm ion bullet"
+	damage = 25
+	wound_bonus = 15
+	armour_penetration = 15
+	var/emp_radius = 0
+
+/obj/projectile/bullet/a762x39/emp/on_hit(atom/target, blocked = FALSE)
+	..()
+	empulse(target, emp_radius, emp_radius)
+	return BULLET_ACT_HIT
+
+/obj/item/ammo_casing/realistic/a762x39/civilian
+	name = "5.6x40mm civilian bullet casing"
+	desc = "A 5.6x40mm civilian-grade surplus bullet casing."
+	extended_desc = "CIVILIAN: Non-military ammunition with a low powder load. Performs worse in every aspect in comparison to its military variant."
+	projectile_type = /obj/projectile/bullet/a762x39/civilian
+
+/obj/projectile/bullet/a762x39/civilian
+	name = "5.6mm surplus bullet"
+	damage = 30
+	wound_bonus = 15
+	armour_penetration = 10
+	wound_falloff_tile = 3
+
+/obj/item/ammo_casing/realistic/a762x39/civilian/rubber
+	name = "5.6x40mm rubber bullet casing"
+	desc = "A 5.6x40mm civilian-grade rubber bullet casing."
+	extended_desc = "RUBBER: Less than lethal ammo. Deals both stamina and brute damage."
+	projectile_type = /obj/projectile/bullet/a762x39/rubber
+	harmful = FALSE
```

Comment:
civilian-facing rubbers should do a bit of brute, and not be usable by pacifists

Reply:
They do. And, afaik, all rubber ammo can be used by pacifists.